taxes_by_invoice/templates/header_macro.html

```diff
@@ -39,9 +49,8 @@ <h1 style="font-size:20px">{% if data['parameters']['jump_page'] %}{{ _('Taxes B
             </tr>
         {% endif %}
         <tr>
-            <td>{% if data['parameters']['parties'] %}{{ _('Parties: ') }}{{ data['parameters']['parties']}}{% else %}{{ _('All Parties') }}{% endif %}</td>
-            <td style="text-align: right;">{% if data['parameters']['include_cancel'] %}{{ _('This document includes cancelled invoices marked with a *') }}{% else %}{{ _('This document not includes cancelled invoices') }}{% endif %}</td>
+            <td colspan="2">{% if data['parameters']['parties'] %}{{ _('Parties: ') }}{{ data['parameters']['parties']}}{% else %}{{ _('All Parties') }}{% endif %}</td>
```

Comment:
You cannot remove the "This document includes...." comment.

Reply:
cea479bafe933384ba3711084880325a7bfdc239